pages/dkp/kommander/2.1/release-notes/2.1.1/index.md

```diff
@@ -77,6 +77,93 @@ The following services and service components have been upgraded to the listed v
 - traefik-forward-auth: 0.3.2
 - velero: 3.1.3
 
+## Known Issues
+
+The following items are known issues with this release.
+
+### Create cert-manager resources on attached clusters with cert-manager pre-installed
+
+If you attach a cluster that already has `cert-manager` installed, you need to manually create the three cert-manager resources, in the yaml file, after attaching your cluster.
+
+For example, [Konvoy-created clusters that are self-managed][konvoy-self-managed] have `cert-manager` already installed to the `cert-manager` namespace.
+
+Verify that your cluster is self-managed by:
```

Comment:
This has the possibility to exist outside of DKP created clusters probably, right? Most likely, if they're using DKP/Kommander 2.1.1, it's going to be a Konvoy created cluster, so this would work.

```suggestion
Verify that your DKP cluster is self-managed by:
```

Reply:
do we actually need to check if a cluster is self-managed, or do we just need to check if cert-manager is already installed?